src/components/Header/Header.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export const Header: React.FC = () => (
+  <header>
+    <nav className="navbar" role="navigation" aria-label="main navigation">
+      <div className="navbar-menu">
+        <div className="navbar-start">
+          <NavLink
+            to="/"
+            className={({ isActive }) => (
+              `navbar-item is-tab ${isActive ? ' is-active' : ''}`
+            )}
```

Comment:
I believe using `classnames` would be a better solution instead of applying ternary with an empty string